src/test/java/uk/gov/pay/api/it/rule/RedisContainer.java

```diff
@@ -128,12 +128,8 @@ void stop() {
     }
 
     public void clearRedisCache() {
-        try {
-            String[] command = {"redis-cli", "FLUSHALL"};
-            String id = docker.execCreate(containerId, command).id();
-            docker.execStart(id);
-        } catch (DockerException | InterruptedException e) {
-            throw new RuntimeException(e);
-        }
+        Jedis jedis = new Jedis(host, port);
+        jedis.flushAll();
```

Comment:
public String flushAll()
Delete all the keys of all the existing databases, not just the currently selected one. This command never fails.
Specified by:
flushAll in interface BasicCommands
Returns:
Status code reply

Reply:
we should probably do some error checking here

Reply:
the return value of `jedis.flushAll()` is `"OK"`...